test_regress/t/t_stack_check.pl

```diff
@@ -16,6 +16,5 @@
 
 execute();
 
-file_grep($Self->{run_log_filename}, qr/.*%Warning: System has stack size/);
 ok(1);
```

Comment:
Replicate this test to t_stack_check_fail.pl with the new -D_VL_TEST_RLIMIT_FAIL=1.  It can also have top_filename("t/t_stack_check.v") so you don't need to copy the tests's .v file.

Then this test remains with this line removed to check that the increase works.

Reply:
Done